docs/openapi/api.yaml

```diff
@@ -125,17 +125,31 @@ paths:
     $ref: './tools-api.yaml#/import-job-progress'
   /tools/import/jobs/{jobId}/result:
     $ref: './tools-api.yaml#/import-job-result'
+  /tools/import/assistant/prompt:
+    $ref: './tools-api.yaml#/import-assistant'
 
 components:
   securitySchemes:
     api_key:
       type: apiKey
       in: header
       name: x-api-key
+      description: "DEPRECATED, use scoped_api_key or ims_key. For read-only operations."
+    scoped_api_key:
+      type: apiKey
+      in: header
+      name: x-api-key
+      description: "For Importer as a Service endpoints"
+    ims_key:
+      type: apiKey
+      in: header
+      name: x-api-key
```

Comment:
Does `ims_key` use the `Authentication` header?

Reply:
@solaris007  ☝️  are the ims_key properties correct?

Reply:
Looks like it indeed uses `authentication`: https://github.com/adobe/spacecat-shared/blob/main/packages/spacecat-shared-http-utils/src/auth/handlers/ims.js#L50

Reply:
Makes sense that it wouldn't be `x-api-key`.

How about its `type`?  **apiKey** doesn't seem correct.  I'm not sure how the `type` mapping is done.